cmd/handler-api.go

```diff
@@ -48,6 +51,40 @@ type apiConfig struct {
 	deleteCleanupInterval       time.Duration
 }
 
+const cgroupLimitFile = "/sys/fs/cgroup/memory/memory.limit_in_bytes"
```

Comment:
Is this file old enough ? Not able to find when it is introduced

Reply:
This present since cgroups was implemented